pelican/plugins/image_process/test_image_process.py

```diff
@@ -571,3 +571,21 @@ def test_is_img_identifiable():
     settings = get_settings(IMAGE_PROCESS_DIR="derivatives")
     path = compute_paths(img, settings, derivative="thumb")
     assert not is_img_identifiable(path.source)
+
+
+def generate_test_images():
+    settings = get_settings()
+    image_count = 0
```

Comment:
It's a detail but no need to increment a variable here, you could juste use `len(TEST_IMAGES)` in your final `print` statement

Reply:
The difference is that `len(TEST_IMAGES)` (or more exactly, `len(SINGLE_TRANSFORMS) * len(TEST_IMAGES)`) gives the number of images that *should* be generated, incrementing like this gives the *actual* number of image generated. Granted, because there doesn't appear to be an error handling, those two numbers should always match if the function doesn't error out. But anyway, that was the logic behind this choice.